VPC/Sg.tf

```diff
@@ -0,0 +1,87 @@
+// Create Security Group for Baiston Host 
+
+resource "aws_security_group" "ssh-security-group" {
+    name = var.name-sg
+    vpc_id = aws_vpc.main.id
+
+    dynamic "ingress" {
+        for_each = var.ports
+        iterator = port
+        content {
+          from_port = port.value
+          to_port = port.value
+          protocol = "tcp"
```

Comment:
What if protocol is also dynamic?

Reply:
In dynamic protocol  we can define multiple protocol  like "TCP",UDP"

Reply:
Resolved